src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -519,6 +520,25 @@ private void InitializeProperties(IEnumerable<string> updatableProperties)
             }
         }
 
+        private bool IsIgnoredProperty(bool isTypeDataContract, PropertyInfo propertyInfo)
```

Comment:
>isTypeDataContract [](http://example.com/codeflow?start=44&length=18)

If the type is marked with [DataContract], but the property is not marked with [DataMember], do we include or ignore it?